### PR TITLE
Added: price-card

### PR DESCRIPTION
**Pull request by KaustubhM1997**
1) Added price card in the cart
2) Mapped featured categories from home page to product listing page
3) Fixed discount % on the product listing

Live preview - https://playful-pegasus-8f02eb.netlify.app/

### CODE REVIEW

File: src/pages/Cart/cart.js [hunk 1]
```diff
@@ -8,8 +8,26 @@ const Cart = () => {
   const {
     state: { cartlist },
   } = useCart();
+  const add = 1 + 2; //test
```
Comment (vijaytembugade) [inline src/pages/Cart/cart.js]: remove unwanted code

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -2,9 +2,13 @@ import "./home.css";
 import { Link } from "react-router-dom";
 import { Card } from "../../components/Home-card/home-card.jsx";
 import { categories } from "../../backend/db/categories";
+import { useProductListing } from "../../contexts/productListing-context";
 
 const Home = () => {
   // const [categoryMen] = categories; //categories is an array of objects // we would do this only if there was one category and pass categoryMen into category below.
+
+  const { dispatch } = useProductListing();
+  // console.log(productDispatch, "there");
```
Comment (vijaytembugade) [inline src/pages/Home/Home.jsx]: remove this comment

File: src/reducers/productReducer.jsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ const productReducer = (state, action) => {
       return { ...state, fastDelivery: action.payload };
     case "CATEGORY":
       const categoryExist = state.category.includes(action.payload); //this stores the payload fired by the user
+      console.log(action.payload, "here");
```
Comment (vijaytembugade) [inline src/reducers/productReducer.jsx]: remove `console.log`

**Review by vijaytembugade — APPROVED** [review]
Looks good to me. Good work

**Review by Varu98 — APPROVED** [review]
Maybe we can add a notification when add to cart or add to wishlist function gets fired. 
Rest is working fine. 👍🏻

**Pull request merged**